frontends/web/src/api/account.ts

```diff
@@ -233,7 +252,15 @@ export interface IBalance {
     incoming: IAmount;
 }
 
-export const getBalance = (code: AccountCode): Promise<IBalance> => {
+export type BalanceResponse = {
```

Comment:
prefix with `T` for consistency with the rest

Reply:
done